packages/tools/src/utilities/polyData/utils.ts

```diff
@@ -1,14 +1,21 @@
 import vtkPolyData from '@kitware/vtk.js/Common/DataModel/PolyData';
+import type { Types } from '@cornerstonejs/core';
+import { vec3 } from 'gl-matrix';
 
 /**
  * Gets a point from an array of numbers given its index
- * @param points array of number, each point defined by three consecutive numbers
- * @param idx index of the point to retrieve
+ * @param points - array of number, each point defined by three consecutive numbers
+ * @param idx - index of the point to retrieve
  * @returns
  */
-export function getPoint(points, idx) {
-  if (idx < points.length / 3) {
-    return [points[idx * 3], points[idx * 3 + 1], points[idx * 3 + 2]];
+export function getPoint(points, idx): Types.Point3 {
```

Comment:
This version is more performant than the previous one, but is compatible.

Reply:
please type the inputs